src/components/Comment/Comment.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { CurrentUserContext } from "../../contexts/CurrentUserContext";
+import { ModalsContext } from "../../contexts/ModalsContext";
+
+import "./Comment.css";
+
+const Comment = ({
+  isLoggedIn,
+  comment,
+  commentName,
+  commentPage,
+  handleDelete,
+  handleCommentToEdit,
+}) => {
+  const currentUser = React.useContext(CurrentUserContext);
+  const modalContext = React.useContext(ModalsContext);
+
+  const handleOpenEditModal = (evt) => {
+    handleCommentToEdit(comment);
+    modalContext.openModals(evt);
+  };
+
+  return (
+    <li className="comment">
+      <div className="comment-wrapper">
```

Comment:
`comment-wrapper` should be an element of `comment`, according to BEM

`comment__wrapper`